source/core/parameters.cc

```diff
@@ -2083,6 +2083,53 @@ namespace Parameters
           "false",
           Patterns::Bool(),
           "A boolean that will force the linear solver to continue even if it fails");
+
+        prm.declare_entry("mg min level",
```

Comment:
Oje. `declare_entry` is still used... I think one should prefer `add_parameter()`.

Reply:
Yeah we still use the old declare / parse approach and not the add parameter. For now let's keep it this way, this is an orthogonal refactoring we can do in the future (endless refactoring is my life)